app/views/articles/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<p><%= notice %></p>
+
+<div class="container m-auto p-4">
+  <h1 class="text-2xl text-center font-bold">Articles</h1>
+<button class="bg-blue-400 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded-full">
+ <%= link_to 'New Article', new_article_path %>
+</button>
+
+</div>
+
+<table class="table-auto">
```

Comment:
tu natomiast wcięcia są niekonswkwentne, czasem to jedna, czasem dwie a czasem cztery spacje :) do poprawki w momencie pracy nad tym plikiem, bo Railsy co prawda się nie czepiają wcięć, ale inne frameworki już mogą bardzomocno, np React :)